AppCenterDataStorage/AppCenterDataStorage/MSPage.h

```diff
@@ -0,0 +1,23 @@
+#import "MSDataStoreError.h"
+#import "MSDocumentWrapper.h"
+#import "MSSerializableDocument.h"
+#import <Foundation/Foundation.h>
```

Comment:
NIT: system import comes at the beginning 

+ 
sort

Reply:
fixed